src/05-invoicing/subscriptions.apib

```diff
@@ -139,6 +139,37 @@
                         + rate: 0.21 (number)
                         + taxable (Money)
                         + tax (Money)
+            + grouped_lines (array)
+                + (object)
+                    + section (object)
+                        + title (string)
```

Comment:
Title can be null on invoices.info so I assume it will be the same here

Reply:
It is [not nullable](https://github.com/teamleadercrm/api/blob/master/src/05-invoicing/invoices.apib#L158) on invoices.info 🤔

Reply:
The documentation is wrong, it can return null.